### PR TITLE
Updates PF dependencies, and Instance Toggle labels

### PR DESCRIPTION
**Pull request by AlexSCorey**
##### SUMMARY
This addresses #11105 and #11106.  Dependency updates were also made in the container.
##### ISSUE TYPE
 - Bugfix Pull Request

##### COMPONENT NAME
 - UI

##### ADDITIONAL INFORMATION
Screen Shot 2021-09-27 at 2 27 42 PM

### CODE REVIEW

**Comment by tiagodread** [comment]
@akus062381 and @InnocentK let's make sure that all the tests are green before merging this PR

**Comment by AlexSCorey** [comment]
@keithjgrant Is doing essentially a total refactor of the Job Output component, and I had a failing unit test there.  So, knowing that he has a whole other unit testing file, and he is refactoring a lot of things I elected to remove a single test assertion from the JobOutput.test.jsx file.

File: awx/ui/src/screens/Job/JobOutput/JobOutput.test.js [hunk 1]
```diff
@@ -112,7 +112,7 @@ describe('<JobOutput />', () => {
       wrapper = mountWithContexts(<JobOutput job={mockJob} />);
     });
     await waitForElement(wrapper, 'JobEvent', (el) => el.length > 0);
-
+    // console.log(wrapper.debug());
```
Comment (kialam) [inline awx/ui/src/screens/Job/JobOutput/JobOutput.test.js:115]: remove this line?

**Review by kialam — APPROVED** [review]
Looks good aside from a stray console log statement. Feel free to merge after deleting it.

**Review by nixocio — APPROVED** [review]
@InnocentK @akus062381 we also need to make sure that visual tests are sane.

changes LGTM.

**Review by tiagodread — APPROVED** [review]
I verified this change last fri, the e2e tests looks good and I was also looking at the visual changes, there was nothing too expressive in terms of visual differences.

**Pull request merged**